### ISSUE TITLE
"fetch is not defined" when calling thumbnail helper in node.js environment

### ISSUE THREAD

**Issue by jbaiter** [IIIF-Commons/vault-helpers#6]
```
ReferenceError: fetch is not defined
    at X.<anonymous> (/home/jojo/projects/pdiiif-ng/monorepo/node_modules/.pnpm/@iiif+vault-helpers@0.9.5/node_modules/@iiif/vault-helpers/dist/cjs/index.js:1:17632)
    at Generator.next (<anonymous>)
    at /home/jojo/projects/pdiiif-ng/monorepo/node_modules/.pnpm/@iiif+vault-helpers@0.9.5/node_modules/@iiif/vault-helpers/dist/cjs/index.js:1:14361
    at new Promise (<anonymous>)
    at x (/home/jojo/projects/pdiiif-ng/monorepo/node_modules/.pnpm/@iiif+vault-helpers@0.9.5/node_modules/@iiif/vault-helpers/dist/cjs/index.js:1:14181)
    at X.fetch (/home/jojo/projects/pdiiif-ng/monorepo/node_modules/.pnpm/@iiif+vault-helpers@0.9.5/node_modules/@iiif/vault-helpers/dist/cjs/index.js:1:17608)
    at X.<anonymous> (/home/jojo/projects/pdiiif-ng/monorepo/node_modules/.pnpm/@iiif+vault-helpers@0.9.5/node_modules/@iiif/vault-helpers/dist/cjs/index.js:1:17409)
    at Generator.next (<anonymous>)
    at /home/jojo/projects/pdiiif-ng/monorepo/node_modules/.pnpm/@iiif+vault-helpers@0.9.5/node_modules/@iiif/vault-helpers/dist/cjs/index.js:1:14361
    at new Promise (<anonymous>)
```

Looks to me like this could be due to a misconfigured rollup build, I can see that `node-fetch` is defined as an external for node.js, but the corresponding `nodeExternal` variable is not used anywhere in the `rollup.config.js`.

**Comment by stephenwf** [comment]
Ah! I think that might be because we are bundling: `@atlas-viewer/iiif-image-api` which is pulling in the browser specific one: https://github.com/atlas-viewer/iiif-image-api/blob/7d3d233919b35f1ca6f220dfc5bd03681f946f66/package.json#L13-L23 

I think it should be fine to mark this as an external dependency (for the ESM build).

**Comment by stephenwf** [comment]
Fix available in `@iiif/vault-helpers@0.9.6`
